src/scenes/Wallet/Balances/index.js

```diff
@@ -384,21 +383,17 @@ export default class MyBalances extends Component {
   };
 
   onWithdrawClicked(event) {
-    const { wallet, walletUnlockDialog } = this.props.store;
```

Comment:
we don't need to show walletUnlock?

Reply:
Showing the wallet unlock is just a fallback now. Decided to not spend the extra time to implement this properly since:
1. We are going light client and using qrypto
2. even for the future Electron version, we unlock at the very beginning before they even load the UI